packages/core/src/App/Containers/app-notification-messages.jsx

```diff
@@ -91,7 +91,8 @@ const AppNotificationMessages = ({
                 setStyle({ top: bounds.top + 8 });
             }
         }
-    }, [is_mt5, notifications_ref, stopNotificationLoading]);
```

Comment:
no need for `stopNotificationLoading` here. it only caused a console error to be thrown in AppNotificationMessages when CFDDashboard was rerendered in an infinite loop before this fix.